src/components/Voting/CongressionalAssemble/index.js

```diff
@@ -1,17 +1,23 @@
 /* eslint-disable react/prop-types */
 import React, { useEffect, useState } from 'react';
-import { useSelector } from 'react-redux';
+import { useDispatch, useSelector } from 'react-redux';
 import { democracySelectors } from '../../../redux/selectors';
 
 import CurrentAssemble from './CurrentAssemble';
 import CandidateVoting from './CandidateVoting';
+import { democracyActions } from '../../../redux/actions';
 
 function CongressionalAssemble() {
+  const dispatch = useDispatch();
   const democracy = useSelector(democracySelectors.selectorDemocracyInfo);
   const [selectedCandidates, setSelectedCandidates] = useState([]);
   const [eligibleUnselectedCandidates, setEligibleUnselectedCandidates] = useState([]);
   const [didChangeSelectedCandidates, setDidChangeSelectedCandidates] = useState(false);
 
+  useEffect(() => {
+    dispatch(democracyActions.getDemocracy.call());
+  }, []);
```

Comment:
We don't want 'dispatch' in dependencies?

Reply:
We do want it in deps, I've forgot it - nice catch!
Shouldn't eslint complain about it...?

Reply:
We need to add rules for this, in our eslint config we don't have it. I can do it, if we want this.